hack/run-perf-profile-creator.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+readonly CURRENT_SCRIPT=$(basename "$0")
+readonly DATA_DIR="./must-gather-content"
+readonly CMD="podman run --entrypoint performance-profile-creator"
+readonly MUST_GATHER_VOL="/must-gather"
+readonly MUST_GATHER_CMD="oc adm must-gather --request-timeout=10m"
+readonly IMG_EXISTS_CMD="podman image exists"
+readonly IMG_PULL_CMD="podman image pull"
+
+PAO_IMG="quay.io/openshift-kni/performance-addon-operator:4.8-snapshot"
+MG_IMG="quay.io/openshift-kni/performance-addon-operator-must-gather:4.8-snapshot"
+MG_TARBALL=""
+ERASE_DATA_DIR="n"
+
+usage() {
+  print "Wrapper usage:"
+  print "  ${CURRENT_SCRIPT} [-h] [-p image] [-m image] [-t path] [-e]  -- [performance-profile-creator flags]"
+  print ""
+  print "Options:"
+  print "   -h                 help for ${CURRENT_SCRIPT}"
+  print "   -p                 Performance Addon Operator image"
+  print "   -m                 Must-gather image"
+  print "   -t                 Must-gather tarball"
+  print "   -e                 erase must-gather cache"
+
+  ${IMG_EXISTS_CMD} "${PAO_IMG}" && ${CMD} "${PAO_IMG}" -h
+}
+
+exit_error() {
+  print "error: $*"
+  usage
+  exit 1
+}
+
+print() {
+  echo  "$*" >&2
+}
+
+check_requirements() {
+  [ -d "${DATA_DIR}" ] || mkdir "${DATA_DIR}"
+
+  ${IMG_EXISTS_CMD} "${PAO_IMG}" || ${IMG_PULL_CMD} "${PAO_IMG}" || \
+      exit_error "Performance Addon Operator image not found"
+
+  [ -z "${MG_TARBALL}" ] || [ -f "${MG_TARBALL}" ] || \
+      exit_error "Must-gather tarball file not found"
+
+  [ -n "${MG_TARBALL}" ] || ${IMG_EXISTS_CMD} "${MG_IMG}" || ${IMG_PULL_CMD} "${MG_IMG}" || \
+      exit_error "Must-gather image not found and must-gather tarball not provided"
+
+  if [ -f "${MG_TARBALL}" ]; then
+      rm -rf "${DATA_DIR:?}"/*
+      tar -zxf "${MG_TARBALL}" --strip-components=1 --directory "${DATA_DIR}" || \
+          exit_error "Can't decompress must-gather tarball"
```

Comment:
I think that:
1. if the user gives the path to already-decompressed tarball, we should consume the data (read only) and leave the system as we found it
2. if the user gives us a tarball and the wrapper does the decompression, it should clean up any temporary directory/files and not leave leftovers

In other words, the top-level command owns the temporary directory/files, must make sure to not leave stuff in the system (unless explicitely asked to).

Reply:
The assumption is the user gives us the tarball or we run the must-gather tool ourselves. Receiving a decompressed must-gather output is not an option, but I can add one.

Reply:
The data_dir is a cache-dir used for both decompression of the tarball - in this case we need to empty it before- , but also we keep the output of a must-gather run, in this case is empty before we run the must-gather or we empty it by passing the -e arg.
We have to "leave stuff" so we will not run must-gather tool every time.
We can make it easier by requiring the tarball and running the must-gather BEFORE starting the wrapper.
It is convenience only approach.

Reply:
Aha, so a flag of "keep" and delete everything by default. Sure.
Regarding of  a new flag that differentiate a local directory vs a tarball, I am not so sure... is becoming "complicated". I don't like a lot of flags.

I want to re-iterate my prev question. Why not asking for the user to provide a must-gather tar.gz as the only input? Everybody knows how to do that and we''ll document it.

Is starting to look like something outside of the wrapper responsibility. 
The user runs must-gather and creates the tarball, passing it as an arg to PPC.
@fromanirh @swatisehgal what do you think?

We can make it smarter based on user needs.

Reply:
Sure, no objections to just having tarball as the only possible input. Like you said can be taken care of by documentation.